frontend/src/components/ide/AIAssistant/components/SettingsPanel.jsx

```
@@ -110,6 +124,119 @@ export function SettingsPanel({ settings, onSave }) {
+                        <div className="flex justify-between text-xs text-muted-foreground mt-1">
+                            <span>Точный (0)</span>
+                            <span>Креативный (2)</span>
+                        </div>
+                    </div>
+
+                    <div>
+                        <label className="text-sm font-medium mb-1 block">Макс. токенов</label>
+                        <Input
+                            type="number"
+                            min="256"
+                            max="32768"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)